ui/job-view/details/PinBoard.jsx

```diff
@@ -179,22 +180,52 @@ export default class PinBoard extends React.Component {
   }
 
   retriggerAllPinnedJobs() {
-    // pushing pinned jobs to a list.
     const jobIds = Object.keys(this.props.pinnedJobs);
-    const plurality = jobIds.length > 1 ? 's' : '';
-
-    JobModel.retrigger(this.$rootScope.repoName, jobIds)
-      .then((resp) => {
-        if (resp.ok) {
-          this.thNotify.send(
-            `Retrigger request sent for ${jobIds.length} pinned job${plurality}`,
-            'success');
-        } else {
-          throw new Error(formatModelError(resp, `Unable to send retrigger${plurality}`));
+
+    if (!this.props.isLoggedIn) {
+      return this.$timeout(this.thNotify.send('Must be logged in to retrigger jobs', 'danger'));
```

Comment:
We shouldn't need ``this.$timeout`` for notify any longer.  So that can be removed.